apps/api/src/services/sendgrid_handler.py

```diff
@@ -36,6 +36,7 @@ class GuestTokenPersonalization(PersonalizationData):
 async def send_email(
     template_id: Literal[Template.CONFIRMATION_EMAIL],
     sender_email: Tuple[str, str],
+    reply_to: Tuple[str, str],
```

Comment:
Issue [repeating]: We'll want to place `reply_to` specifically after `receiver_data` since that's how we order the arguments in our function calls.